aspnetcore/signalr/get-started-signalr-core.md

```diff
@@ -0,0 +1,106 @@
+---
+uid: signalr/get-started-signalr-core
+title: Get started with SignalR on ASP.NET Core
+author: rachelappel
+ms.author: rachelap
+description: In this tutorial, you create an app using SignalR for ASP.NET Core.
+manager: wpickett
+ms.date: 02/23/2018
+ms.topic: tutorial
+ms.technology: dotnet-signalr
+ms.prod: aspnet-core
+ms.custom: mvc
+---
+# Tutorial: Get started with SignalR for ASP.NET Core
+
+By [Rachel Appel](https://twitter.com/rachelappel)
+
+This tutorial teaches the basics of building a real-time app using SignalR for ASP.NET Core.
+
+   ![Solution](get-started-signalr-core/_static/signalr-get-started-finished.png)
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/signalr/get-started-signalr-core/sample/) ([how to download](xref:tutorials/index#how-to-download-a-sample))
+
+This tutorial demonstrates the following SignalR development tasks:
+
+> [!div class="checklist"]
+> * Create an ASP.NET Core web app with a SignalR hub.
```

Comment:
Now that I'm reading the whole list, I think the "SignalR hub" part becomes redundant with the second item. Maybe just:

* Create an ASP.NET Core Web App
* Create a SignalR Hub to push content to clients
* Use the SignalR JavaScript library to send messages and display updates from the hub.